test/unit/test.rdf.js

```diff
@@ -333,6 +333,21 @@ describe("lib/rdf", function() {
       expect(firefox.childNodes[5].tagName).to.be.equal("em:maxVersion");
       expect(firefox.childNodes[5].childNodes[0].data).to.be.equal(MAX_VERSION);
     });
+
+    it("passes through raw GUIDs as targetApplication", function() {
+      var xml = setupRDF({ engines: {
+        "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}": "*"
```

Comment:
I don't think this is such a good test because it's identical to the Firefox GUID. How about just something like `{xxxxxxxx-c20a-464f-9b0e-yyyyyyyyyyyy}` ?